### ISSUE TITLE
Raid ID 2105 Client crash, No GM's around server to help

### ISSUE THREAD

**Issue by Shagnastee** [dalaranwow/dalaran-wow#5934]
==============================================================================
World of WarCraft (build 12340)

Exe:      C:\Users\Downloads\World of Warcraft - WoTLK\Wow.exe
Time:     Nov  9, 2020  4:17:03.783 PM

------------------------------------------------------------------------------

This application has encountered a critical error:

ERROR #132 (0x85100084) Fatal Exception
Program:	C:\Users\Downloads\World of Warcraft - WoTLK\Wow.exe
Exception:	0xC0000005 (ACCESS_VIOLATION) at 0023:008E47C2

The instruction at "0x008E47C2" referenced memory at "0x00000008".
The memory could not be "read".

WoWBuild: 12340
Realm: Algalon - Main Realm [37.187.164.32:8086]
Local Zone: Ulduar
Local Player: Shagnastee, 0000000000182D0D, (2141.14,-258.269,419.828)
Attacking Target: Hodir, F13000806100073E, (2136.1,-259.245,419.846)
Locked Target: Hodir, F13000806100073E, (2136.1,-259.245,419.846)
Last Enemy Target: Hodir, F13000806100073E, (2136.1,-259.245,419.846)
Total lua memory: 22844KB
Current Addon: (null)
Current Addon function: UNKNOWN
Current Addon object: (null)
Add Ons: GearScoreLite HealBot Recount DBM-Core DBM-Ulduar 
Settings: 
SET locale "enGB"
SET realmList "logon.dalaran-realmlist.org"
SET hwDetect "0"
SET gxResolution "1280x1024"
SET gxRefresh "60"
SET gxMultisampleQuality "0.000000"
SET gxFixLag "0"
SET videoOptionsVersion "3"
SET movie "0"
SET Gamma "1.000000"
SET readTOS "1"
SET readEULA "1"
SET readTerminationWithoutNotice "1"
SET showToolsUI "1"
SET Sound_OutputDriverName "System Default"
SET Sound_MusicVolume "0.40000000596046"
SET Sound_AmbienceVolume "0.60000002384186"
SET farclip "1277"
SET specular "1"
SET groundEffectDensity "24"
SET projectedTextures "1"
SET mouseSpeed "1"
SET accounttype "LK"
SET realmName "Algalon - Main Realm"
SET gameTip "5"
SET gxWindow "1"
SET gxMaximize "1"
SET lastCharacterIndex "2"

**Comment by MalkaviaDW** [comment]
Hello!

Try to download a new client or repair the current one

**Comment by Shagnastee** [comment]
> 
> 
> Hello!
> 
> Try to download a new client or repair the current one. Open an ingame ticket if you need any help.

I opened my first ticket, GM asked that i open a new one when the run was done.  I opened the second ticket.  Had GM Azerotth message me two days later asking that i link the missing achievement, i was afk at the time, responded to him, he was afk and then logged out.
Do i need to open another ticket?

The client works fine, it had a issue with Hodir and Thorim on that run, but it recovered on Thorim and i got credit for the kill.
